hcl2template/types.hcl_post-processor.go

```diff
@@ -39,6 +39,8 @@ func (p *HCL2PostProcessor) HCL2Prepare(buildVars map[string]interface{}) error
 				buildValues[k] = cty.NumberIntVal(v)
 			case uint64:
 				buildValues[k] = cty.NumberUIntVal(v)
+			case bool:
+				buildValues[k] = cty.BoolVal(v)
 			default:
```

Comment:
Is the name of the failed variable in the logs as well somewhere above this return? If not it would be a nice addition to the error message.